AllReadyApp/Web-App/AllReady.UnitTest/Areas/Admin/Controllers/SiteAdminControllerShould.cs

```diff
@@ -31,11 +34,15 @@ public void IndexReturnsCorrectViewModel()
             await TaskFromResultZero;
         }
 
-        [Fact(Skip = "NotImplemented")]
-        public async Task DeleteUserHasHttpGetAttribute()
+        [Fact]
+        public void DeleteUserHasHttpGetAttribute()
         {
-            //delete this line when starting work on this unit test
-            await TaskFromResultZero;
+            var mediator = new Mock<IMediator>();
+            var controller = new SiteController(null, null, mediator.Object);
```

Comment:
I did not intend to delete js files. I'm not sure what happened there. I'll be sure to add those back in and make the change that @mgmccarthy mentioned.